RELEASE.md

```diff
@@ -0,0 +1,17 @@
+# Release flow
+
+There are two ways how release and/or deploy works.
+
+## Steps
+
+1. You push to the main branch and it's automated deployed to various repositories to use as latest release version with hash commit.
+2. You push new tag so it means you want to create release, but the flow of deployment is almost the same as you push to the main.
+
+## Version
+
+You should to update current version in the file `APP_VERSION` before creating the release.
+
+Please, remember that odd numbers are for development and even is for releases.
+
+0.3.1 or 0.5.15 – pre-release or dev version
+0.3.0 or 0.5.10 – release version
```

Comment:
```suggestion
# Release flow

There are two ways how release and/or deploy works.

## Steps

1. You push to the main branch and it's automatically deployed to various APT/YUM etc. repositories on https://repo.manticoresearch.com/ to be used as the latest dev version with a hash of your commit.
2. You push a new tag meaning you want to create a release. Otherwise the deployment flow is almost the same as when you push to the main branch.

## Version

You should update current version in the file `APP_VERSION` before creating a release.

Please, remember that odd numbers are for development and even are for releases.

0.3.1 or 0.5.15 – pre-release or dev version
0.3.0 or 0.5.10 – release version
```